docs/spec/manifest-v2-2.md

```diff
@@ -0,0 +1,255 @@
+# Image Manifest Version 2, Schema 2
+
+This document outlines the format of of the V2 image manifest, schema version 2.
+The original (and provisional) image manifest for V2 (schema 1), was introduced
+in the Docker daemon in the [v1.3.0
+release](https://github.com/docker/docker/commit/9f482a66ab37ec396ac61ed0c00d59122ac07453)
+and is specified in the [schema 1 manifest definition](./manifest-v2-1.md)
+
+This second schema version has two primary goals. The first is to allow
+multi-architecture images, through a "fat manifest" which references image
```

Comment:
I don't really think `docker build` makes sense for multi-arch.

> A manifest can either have a set of layers, or a set of manifests. However, that would allow having manifests of manifests of manifests.

The original design from this phase said manifests enumerated resources. Those resources could be layers or other manifests. I am not sure why we departed from this.